src/app/template-library/template-library.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="ion-page" style="background: #ffffff">
+<ion-split-pane class="template-library-container" contentId="main">
+  <ion-menu contentId="main" style="border: none">
+    <div class="search-home-container">
+      <ion-searchbar color="light" class="template-search-bar"></ion-searchbar>
+    </div>
+    <h6 class="menu-header">Categories</h6>
+    <ion-content>
+      <ng-container *ngFor="let category of categories">
+        <p class="cursor-pointer" (click)="viewCategory(category)">{{ category.name }}</p>
```

Comment:
we can use `ion-list` and `ion-item` for this list, so that it looks better

Reply:
I actually tried with these components before and decided not to use them because they add margin, padding and borders to the list items which we don't want. I found I was just fighting the styling. This is why I decided on using `<p>` tags. 

Not sure if this might cause a problem with mobile though?

Reply:
It looks like this when I use the ionic tags

![Practera](https://user-images.githubusercontent.com/8277633/113113318-e530fb00-9255-11eb-8e76-7c9d1e58beab.png)

Reply:
it looks fine for me besides the underline which can be removed with `lines="none"` on `ion-item`.
We can also change the padding with CSS for them. https://ionicframework.com/docs/api/item here is the documentation of it. 
Generally, we should try to use ionic/bootstrap components as much as we can so that we don't need to worry about a lot of things ourselves like being responsive. etc.